app/controllers/miq_ae_tools_controller.rb

```diff
@@ -194,44 +194,55 @@ def review_import
   def retrieve_git_datastore
```

Comment:
@mkanoor Can we break this method down into a few smaller methods parts.  It has grown to be a rather large method with these changes.